hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/TaskletExecutionService.java

```diff
@@ -281,7 +281,7 @@ public void run() {
 
         @Probe(name = "taskletCount")
         private final CopyOnWriteArrayList<TaskletTracker> trackers;
-        @Probe
+        @Probe(name = "iterationCount")
```

Comment:
are these not in `MetricNames`?

Reply:
I've only added those metric names to `MetricNames` which are retrievable via API, so only the job specific ones. 

Do you want me to add them? 

Actually there are a lot more non-job specific metrics we define and which weren't mentioned in the manual... They are specific to certain processors, but still, they are there. Should I add those too?

Reply:
I don't think we need to mention the processor-specific ones for now, but the other general metrics should be added to the manual since they're still retrievable via JMX

Reply:
Ok.